elastic/kibana
# [Dev Tools] Ability to disable app

## Summary

Adds the ability to disable a dev tool app in the tabbed content, this brings the DevTools API in line with the management section app API.

### How to test

See any of the fake disables in `42dcc75` and do the reverse and you should see the disabled tab in action.

### Screenshots

<img width="1021" alt="Screenshot 2020-04-08 at 15 40 25" src="https://user-images.githubusercontent.com/8155004/78792784-da7cb680-79b1-11ea-8f91-85c29ae76c2a.png">


### Gif

![cannotnavdisabled](https://user-images.githubusercontent.com/8155004/78792807-e36d8800-79b1-11ea-985b-e77fc4c03738.gif)
